### ISSUE TITLE
Help not always working for dotnet solution

### ISSUE THREAD

**Issue by michaelgwelch** [microsoft/vscode#103289]
Issue Type: <b>Bug</b>

The system info below (captured by Report Issue) doesn't make it clear (to me) that this is macOS Catalina 10.15.6. 

Steps to Reproduce:

1. Use dotnet to create projects.
2. Open in Code
3. Sometimes syntax coloring and help are working and sometimes not.

In this repo: https://github.com/michaelgwelch/garbage 

The Console project has help, the ClassLib doesn't. By which I mean, if I hover over a method in the Console app I get pop up help. If I hover over the method in ClassLib I just get type information.

ClassLib

<img width="622" alt="image" src="https://user-images.githubusercontent.com/826112/88443149-cb910700-cddc-11ea-9dbe-8a5b667c384d.png">

Console:

<img width="619" alt="image" src="https://user-images.githubusercontent.com/826112/88443158-d481d880-cddc-11ea-9385-974c05d5a35c.png">

Does this issue occur when all extensions are disabled?: Yes

I uninstalled every single extension (except OmniSharp) and restarted VS Code

<img width="360" alt="image" src="https://user-images.githubusercontent.com/826112/88443200-ef544d00-cddc-11ea-8f9f-c37992184882.png">

VS Code version: Code 1.47.2 (17299e413d5590b14ab0340ea477cdd86ff13daf, 2020-07-15T18:18:50.054Z)
OS version: Darwin x64 19.6.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8210Y CPU @ 1.60GHz (4 x 1600)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|5, 12, 9|
|Memory (System)|16.00GB (1.80GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (1)</summary>

Extension|Author (truncated)|Version
---|---|---
csharp|ms-|1.22.1

</details>

**Comment by michaelgwelch** [comment]
I think I discovered the issue. I switch to Visual Studio for Mac and noticed one of my class libs still wasn't showing help. So I added a new class lib to the solution.

I then look at the differences in their dependencies in the Project Explorer. What I saw was different. One listed a Framework and the other listed an SDK. I opened up the project files and saw they were identical expect one was netstandard2.0 and the other was netstandard2.1. The fix was to upgrade the project that didn't have help to net standard2.1. Then help started to work.

Since it impacts both IDEs I guess this may be a dotnet core issue?
